CHANGELOG.md

```diff
@@ -24,6 +24,13 @@
     - FragmentProps
     - StrictModeProps
     - ReduxProviderProps (for extension, mix in ReduxProviderPropsMixin instead)
+    - ProviderProps (factory: `Context.Provider`)
+    - ConsumerProps (factory: `Context.Consumer`)
+- The following props are now required:
```

Comment:
I think this is all of them, but we'll do another pass to fill in any missing changelog entries before we release